<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagementConnectClient</class>
 <widget class="QDialog" name="ManagementConnectClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bodyWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>750</width>
     <height>350</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="tipLabel_1">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>220</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>输入序号主动</string>
    </property>
   </widget>
   <widget class="QLabel" name="tipLabel_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>断开与客户端的连接：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="disconnectLineEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>260</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>241</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>断开连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="tipLabel_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>290</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>(请注意检查断开目标是否正确）</string>
    </property>
   </widget>
   <widget class="QLabel" name="clientsCountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>当前连接数：0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
